cupy/indexing/generate.py

```diff
@@ -252,7 +252,57 @@ def ix_(*args):
 # TODO(okuta): Implement ravel_multi_index
 
 
-# TODO(okuta): Implement unravel_index
+def unravel_index(indices, dims, order='C'):
+    """Converts a flat index or array of flat indices into a tuple of coordinate arrays.
+
+    Args:
```

Comment:
Please describe accepted types (or values) of arguments.

Reply:
I followed the style of other functions' docs. For example, the docs of `ix_` does not have type description. I thought that was cupy style.

Reply:
Even though it's missing in some functions, we should document it as much as possible.

Reply:
OK.